### DISCUSSION TITLE
Failed to connect to codespace

### DISCUSSION THREAD

**Discussion by MoonLGH** [community/community#43507] [Codespaces]
### Select Topic Area

Product Feedback

### Body

i was getting Failed to connect to codespace, and i see the comments on #43016 and seems like clearing browser cache fix them, but i did that just now and this is now what i got
![image](https://user-images.githubusercontent.com/66781351/211067687-55c7e3d6-08d2-4acf-a79d-30bad37af9b6.png)

**Comment by Face248444** [comment]
Same for me
